src/PaymentGateways/PayPalCommerce/onBoardingRedirectHandler.php

```diff
@@ -25,17 +25,24 @@ class onBoardingRedirectHandler {
 	 */
 	private $webhooksRepository;
 
+	/**
+	 * @var MerchantDetails
```

Comment:
@JasonTheAdams  Please add `@since` tag.

Reply:
Resolved in 3471eb5